app/src/main/java/com/conupods/Settings/SettingsPersonalActivity.java

```diff
@@ -0,0 +1,60 @@
+package com.conupods.Settings;
+
+import android.content.Context;
+import android.content.Intent;
+import android.content.SharedPreferences;
+import android.os.Bundle;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.TextView;
+
+import com.conupods.OutdoorMaps.View.MapsActivity;
+import com.conupods.R;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+public class SettingsPersonalActivity extends AppCompatActivity {
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        setContentView(R.layout.settings_page_personal);
+
+        SharedPreferences preferences = getApplicationContext().getSharedPreferences("Preferences", Context.MODE_PRIVATE);
+        SharedPreferences.Editor prefEdit = preferences.edit();
+
+        Button done = findViewById(R.id.done1);
+        done.setOnClickListener(view -> {
+            startActivityIfNeeded(new Intent(SettingsPersonalActivity.this, MapsActivity.class).setFlags(Intent.FLAG_ACTIVITY_REORDER_TO_FRONT), 0);
+        });
+
+        Button defaultPage = findViewById(R.id.toggle1_1);
+        defaultPage.setOnClickListener(view -> {
+            startActivityIfNeeded(new Intent(SettingsPersonalActivity.this, SettingsActivity.class).setFlags(Intent.FLAG_ACTIVITY_REORDER_TO_FRONT), 0);
+        });
+
+        Button infoPage = findViewById(R.id.toggle1_2);
+        infoPage.setOnClickListener(view -> {
+            startActivityIfNeeded(new Intent(SettingsPersonalActivity.this, SettingsInfoActivity.class).setFlags(Intent.FLAG_ACTIVITY_REORDER_TO_FRONT), 0);
+        });
+
+        EditText myAccount = findViewById(R.id.email);
+        myAccount.setOnFocusChangeListener((view, hasFocus) -> {
+            //Implement account connection
+            if(!hasFocus)
+                prefEdit.putString(String.valueOf(myAccount.getId()), String.valueOf(myAccount.getText())).apply();
+        });
```

Comment:
this should also be connected too the google calendar like if i connect to an email should i also be connected too the google calendar?

Reply:
That's something I was not certain about. So you mean if somebody enters an email in the top box, then there should be an automatic connection with google calendar and display connected in the bottom box?

Reply:
I imagine so as that is the whole purpose of connecting too the email. but you for sure shouldnt be able to connect too it without an email